cli/command/stack/common.go

```diff
@@ -67,6 +68,13 @@ func getStackConfigs(
 	apiclient client.APIClient,
 	namespace string,
 ) ([]swarm.Config, error) {
+	version, err := apiclient.ServerVersion(ctx)
+	if err != nil {
+		return []swarm.Config{}, err
+	}
+	if versions.LessThan(version.APIVersion, "1.30") {
+		return []swarm.Config{}, nil
+	}
```

Comment:
I wonder if we should still print a warning (otherwise people may not be aware an option was ignored)